mailer/CHANGELOG.md

```diff
@@ -1,229 +0,0 @@
-<a name="1.81.0"></a>
```

Comment:
Let's keep to remember which features landed. We can move it to the `docs/` dir though, this way we can get rid of the `mailer` dir